notify/grafana_alertmanager.go

```diff
@@ -157,8 +158,17 @@ type Configuration interface {
 
 	Hash() [16]byte
 	Raw() []byte
+	PipelineAndSateTimestampsMismatchAction() PipelineAndSateTimestampsMismatchAction
 }
 
+type PipelineAndSateTimestampsMismatchAction string
+
+var (
+	Disabled     PipelineAndSateTimestampsMismatchAction = "disabled"
+	LogOnly      PipelineAndSateTimestampsMismatchAction = "log-only"
+	StopPipeline PipelineAndSateTimestampsMismatchAction = "stop-pipeline"
```

Comment:
Considering this is in `notify` package, it might make sense to be more specific in the variable names. E.g. Following the same pattern as the type (or shorter might be better). WDYT?
```suggestion
	PipelineAndStateTimestampsMismatchDisabled     PipelineAndStateTimestampsMismatchAction = "disabled"
	PipelineAndStateTimestampsMismatchLogOnly      PipelineAndStateTimestampsMismatchAction = "log-only"
	PipelineAndStateTimestampsMismatchLogStop PipelineAndStateTimestampsMismatchAction = "stop-pipeline"
```

Reply:
Agree